src/python/espressomd/polymer.py

```diff
@@ -75,21 +46,21 @@ def linear_polymer_positions(**kwargs):
     seed : :obj:`int`, required
         Seed for the RNG used to generate the particle positions.
     bond_angle : :obj:`float`, optional
-        If set, this parameter defines the angle between adjacent bonds
-        within a polymer.
-    start_positions : array_like :obj:`float`.
+        Angle in radians between adjacent bonds within a polymer.
+        Must be in the range :math:`[0, \\pi]`. If omitted, random
+        angles are drawn.
+    start_positions : (N, 3) array_like :obj:`float`, optional
         If set, this vector defines the start positions for the polymers, i.e.,
         the position of each polymer's first monomer bead.
-        Here, a numpy array of shape (n_polymers, 3) is expected.
+        Here, a numpy array of shape ``(n_polymers, 3)`` is expected.
     min_distance : :obj:`float`, optional
-        Minimum distance between all generated positions. Defaults to 0
```

Comment:
I liked the default statement

Reply:
Isn't it redundant, considering Sphinx displays `min_distance=0.` in the function parameters list?

Reply:
didn't see that, nvm then